<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddonsWidget</class>
 <widget class="QWidget" name="AddonsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Arial&quot;;
QWidget#AddonsWidget{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QFrame" name="topFrame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>402</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#topFrame {
	background-color: #f1f1f1;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#searchButton{
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.00966184 rgb(45, 45, 45), stop:0.637681 rgb(75, 75, 75));
    color: white;
    border: 1px solid black;
    border-radius: 5px;
	image: url(:/img/search.png);
}
QPushButton#searchButton:hover{
	background-color: black;
}
QPushButton#searchButton:pressed{
	background-color: rgb(76,76,76);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/box.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gray;
font: 8pt ;</string>
    </property>
    <property name="text">
     <string>Results:</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultCountLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gray;
font: 8pt ;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>102</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
	border: none;
}
QTableView::item:selected {
    background-color: rgb(45, 45, 45);
    color: #AAAAAA;
	border:none;
 }
 QHeaderView::section {
     	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.00966184 rgb(45, 45, 45), stop:0.637681 rgb(75, 75, 75));
     color: white;
     padding-left: 4px;
     border: 1px solid rgb(45, 45, 45);
 }

 QHeaderView::section:checked {
     background-color: black;
 }</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditKeyPressed</set>
   </property>
  </widget>
  <widget class="QFrame" name="lowFrame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>290</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QFrame#lowFrame {
	background-color: #f1f1f1;

}
color: gray;
font: 8pt ;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Addon name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Category:</string>
    </property>
   </widget>
   <widget class="QLabel" name="categoryLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="hitsLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rating:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ratingLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#updateButton {

		background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.00966184 rgb(45, 45, 45), stop:0.637681 rgb(75, 75, 75));
    color: white;
    border: 1px solid black;
    border-radius: 5px;
	font:  10pt &quot;Arial&quot; ;
	font-weight: bold;
	image: url(:/img/update.png);
}
QPushButton#updateButton:hover{
	background-color: black;
}
QPushButton#updateButton:pressed{
	background-color: rgb(76,76,76);
}
QPushButton#updateButton:hover {
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="downloadFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	color: #AAAAAA;
}
QFrame#downloadFrame {
	background-color: rgb(45, 45, 45);
	color: #AAAAAA;
	border: 1px solid black;
}
	</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="versionCombo">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>71</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
	background-color: transparent;
	border:none;
	padding:1px;
	text-align: right;
	color: #AAAAAA;
}

QComboBox::drop-down
{
	border-width:1px;
}

QComboBox::down-arrow
{
	image: url(:/img/drop-down.png);
}
QComboBox:hover{
    background-color: transparent;
	background-color: rgb(76, 76, 76);
}
</string>
     </property>
    </widget>
    <widget class="QComboBox" name="filenameCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>1</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
	background-color: transparent;
	border:none;
	padding:1px;
	text-align: right;
	color: #AAAAAA;
}

QComboBox::drop-down
{
border-width:1px;
}

QComboBox::down-arrow
{
	
	image: url(:/img/drop-down.png);
}
QComboBox:hover{
	background-color: transparent;
	background-color: rgb(76, 76, 76);
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="downloadsLabel">
     <property name="geometry">
      <rect>
       <x>293</x>
       <y>8</y>
       <width>57</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>351</x>
       <y>0</y>
       <width>30</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	
	background-image: url(:/img/download.png);

	background-color: transparent;
	border: none;
border: 1px solid black;
}

QPushButton:hover{
	background-color: rgb(76, 76, 76);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="downloadsLabel_2">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>8</y>
       <width>55</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Downloads:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
